source/guides/network/running-a-symbol-node-manually.rst

```diff
@@ -0,0 +1,194 @@
+
+2. Create a directory to store the data files and **run the database**:
+
+   .. code-block:: bash
+
+      mkdir dbfiles
+      mongod --dbpath=dbfiles --wiredTigerCacheSizeGb 2 --bind_ip 127.0.0.1
+
+   Keep ``mongod`` running for as long as your node runs.
+
+3. **Build indices** to optimize database accesses.
+
+   The ``catapult-server`` repository you cloned to build the server in the first step contains `a folder with mongo scripts <https://github.com/nemtech/catapult-server/blob/main/scripts/mongo>`__. Run this one:
+
+   .. code-block:: bash
+
+      mongo 127.0.0.1/catapult < ../../scripts/mongo/mongoDbPrepare.js
+
+   Without this step database performance will be unacceptably slow.
+
+Run the broker
+===================
+
+4. **Edit** ``resources/config-database.properties`` and point ``databaseUri`` to ``127.0.0.1:27017``:
+
+   .. code-block:: ini
+
+      [database]
+  
+      databaseUri = mongodb://127.0.0.1:27017
+      databaseName = catapult
+
+5. **Run the broker**:
+
+   Move into the ``bin`` directory and run:
+
+   .. code-block:: bash
+
+      ./catapult.broker ..
+
+   Keep the broker running for as long as your node runs.
+
+Build and run the REST gateway
+==============================
+
+6. **Clone** the ``catapult-rest`` repository and **install its dependencies**:
+
+   .. code-block:: bash
+
+      git clone https://github.com/nemtech/catapult-rest.git
+      cd catapult-rest
+      npm install -g yarn
+      ./yarn-setup.sh
+
+7. **Build** the REST gateway:
+
+   .. code-block:: bash
+
+      cd rest
+      yarn build
+
+8. **Configure** the REST gateway:
+
+   All configuration is stored in the ``resources/rest.json`` file.
+   
+   **Edit this file** to make sure that the following properties point to the right files:
+
+   .. csv-table::
+      :header: "Property", "File", "Default location"
+      :widths: 35,35,30
+      :delim: ;
+
+      ``tlsClientCertificatePath``; ``node.crt.pem``; ``_build/certificate/``
+      ``tlsClientKeyPath``; ``node.key.pem``; ``_build/certificate/``
+      ``tlsCaCertificatePath``; ``ca.cert.pem``; ``_build/certificate/``
+      ``networkPropertyFilePath``; ``config-network.properties``; ``_build/resources/``
+      ``nodePropertyFilePath``; ``config-node.properties``; ``_build/resources/``
+
+   For example:
+
+   .. code-block:: json
+
+      {
+         "host": "127.0.0.1",
+         "port": 7900,
+         "timeout": 1000,
+         "tlsClientCertificatePath": "~/catapult-server/_build/certificate/node.crt.pem",
+         "tlsClientKeyPath": "~/catapult-server/_build/certificate/node.key.pem",
+         "tlsCaCertificatePath": "~/catapult-server/_build/certificate/ca.cert.pem",
+         "networkPropertyFilePath": "~/catapult-server/_build/resources/config-network.properties",
+         "nodePropertyFilePath": "~/catapult-server/_build/resources/config-node.properties"
```

Comment:
the use of "~/" makes the gateway exit with
```
error: uncaughtException: ENOENT: no such file or directory, open '~/mm-s/main/catapult-server/_build/certificate/node.crt.pem'
```
however, if the path is fully expanded as in
```
   "tlsClientCertificatePath": "/home/marc-os/mm-s/main/catapult-server/_build/certificate/node.crt.pem",
    "tlsClientKeyPath": "/home/marc-os/mm-s/main/catapult-server/_build/certificate/node.key.pem",
    "tlsCaCertificatePath": "/home/marc-os/mm-s/main/catapult-server/_build/certificate/ca.cert.pem",
    "networkPropertyFilePath": "/home/marc-os/mm-s/main/catapult-server/_build/resources/config-network.properties",
    "nodePropertyFilePath": "/home/marc-os/mm-s/main/catapult-server/_build/resources/config-node.properties"

```
It does work well

Reply:
Good to know!